flowcraft/generator/process_details.py

```diff
@@ -71,6 +71,22 @@ def procs_dict_parser(procs_dict):
                     arg_msg = "None"
                 else:
                     arg_msg = ", ".join(dict_proc_info[info])
+            elif isinstance(dict_proc_info[info], dict):
+                # this is used for the "directives", which is a dict
+                if len(dict_proc_info[info]) == 0:
+                    # if dict is empty then add None to the message
+                    arg_msg = "None"
+                else:
+                    # otherwise fetch all template names within a component
+                    # and all the directives for each template to a list
+                    list_msg = ["\n      {}: {}".format(
```

Comment:
Are those spaces indentation? If yes, I would favor using `\t` characters, as it is easier to understand how many indentation levels we're specifying. If we want to add other stuff to display with this option and want to use indentation, it's also easier.

Reply:
Changed this to `\t` instead of `   `.